e2e/test/netstandard20/ProvisioningE2ETests.cs

```diff
@@ -1241,10 +989,8 @@ public static string ComputeDerivedSymmetricKey(byte[] masterKey, string registr
             }
         }
 
-        private bool ImplementsWebProxy(Client.TransportType transportProtocol)
+        public static bool ImplementsWebProxy(Client.TransportType transportProtocol)
```

Comment:
Why public?

Reply:
It is used by the reprovisioning test suite now, too